staffeli_nt/upload.py

```diff
@@ -97,7 +98,7 @@ def grade(submission, grade, feedback, dry_run=True):
                 continue
 
             path = os.path.join(root, name)
-            # print(path)
+            print(path)
```

Comment:
This looks like some (of my) debug print. If you comment it back out I'm happy to merge

Reply:
I think it is some of your debug print. 
It has been very helpful for me, when I've typed in e.g. smileys `:)` in feedback and forgot about yaml-restrictions. 
It will print the path of the yaml-file that caused an error, right before the yaml-parsing exception is thrown that lists offending linenumber(s).
That's why I re-introduced it.

It **should** be handled in a nicer way though. 
I'll remove it and think of a better solution. Perhaps with a --verbose flag.